src/metabase/pivot/core.cljc

```diff
@@ -151,7 +161,7 @@
   (let [col-and-row-indexes (into (vec col-indexes) row-indexes)]
     (reduce
      (fn [acc row]
-       (let [value-key  (select-indexes row col-and-row-indexes)
+       (let [value-key  (map json-roundtrip (select-indexes row col-and-row-indexes))
```

Comment:
Do you actually need to map `json-roundtrip` here? I think calling it on an array/vector would just work.

Reply:
good call